### PR TITLE
fix: Tailwind the Add User page + cypress tests for user feature

### LINKED ISSUE #4320: Create a new user

### LINKED ISSUE #4321: update the existing user profile and link facility

### LINKED ISSUE #4322: Tailwind the Add User page
# Bug

## Describe the problem

- The `Add User` page uses MUI components

### To Reproduce

1. Go to https://care.coronasafe.in/users/add

## Expected behavior

- Tailwind the `Add user` page

## Screenshots

image

### LINKED ISSUE #4491: View password option
**Is your feature request related to a problem? Please describe.**
most of the users of care are people that do not use computers very often and are not very good a typing. In most cases, after they type in passwords they are unsure of accuracy. They erase it and type it out multiple times before submitting

**Describe the solution you'd like**
Add a "view password" option on all password fields to make input visible. 
Something like the below image
Screenshot 2023-01-04 at 1 47 53 PM

### PR DESCRIPTION
**Pull request by cp-Coder**
# Bug Fix

## Proposed Changes

- Tailwind the `Add User` page
- Fixes cypress test for `users`

## Associated Issue

- Fixes #4322
- Fixes #4320
- Fixes #4321
- Fixes #4491 

## Screenshots

image
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug/test a new feature.
- [ ] Update product [documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care)
- [ ] Ensure that UI text is kept in I18n files
- [x] Prep the screenshot or demo video for the changelog entry, and attach it to issue
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by rithviknishad** [comment]
@cp-Coder marking it as hold, since we will now have `FormField` compliant components coming, for example `SelectFormField` shall be used in place of `SelectMenuV2`.

`FormField` components have similar design and API to that of `TextFormField` which you have used so far in many places.

**Comment by rithviknishad** [comment]
Coming via #4309

File: src/Components/Form/FormFields/FormField.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const FormField = (props: {
     props.props;
 
   return (
-    <div className={className}>
+    <div className={className} id={id}>
```
Comment (rithviknishad) [inline src/Components/Form/FormFields/FormField.tsx]: The ID is supposed to be used by the actual field component. Else we'll end up having the same id for the div and the component wrapped by the FormField

```suggestion
    <div className={className}>
```

**Comment by rithviknishad** [comment]
unholding coz just realized its a plug-and-play replacement anyways which can be done later too :)

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by khavinshankar** [comment]
@cp-Coder do fix the failing cypress tests

**Comment by khavinshankar** [comment]
@cp-Coder seems there are failing tests, please do check

**Comment by khavinshankar** [comment]
@cp-Coder do check the cypress tests

**Comment by aparnacoronasafe** [comment]
@nihal467 @rithviknishad When are we expecting this to go live?

**Pull request merged**